app/models/agents/twitter_user_agent.rb

```diff
@@ -57,12 +63,12 @@ def default_options
         'username' => 'tectonic',
         'include_retweets' => 'true',
         'exclude_replies' => 'false',
-        'expected_update_period_in_days' => '2'
+        'expected_update_period_in_days' => '2',
+        'choose_home_time_line' => 'false'
       }
     end
 
     def validate_options
-      errors.add(:base, "username is required") unless options['username'].present?
```

Comment:
I put in a solution for this section. I am aware that options is in json structure. I am not sure if I did it correctly. My last test is failing and its linked to this. When I run rspec I am getting Validation failed: username is required.